### ISSUE TITLE
Gatsby "Error: Interface field Craft_FormInterface.fields expected but Craft_Form does not provide it."

### ISSUE THREAD

**Issue by webrgp** [verbb/formie#617]
**Description**

Getting an error using the Craft Gatsby Plugin. I think is related to #610 issue.

**Steps to reproduce**
Same as #610 

### Form settings
Please pick from the options below.

**Additional info**
- Plugin version: 1.4.22
- Craft version: 3.7.17.1
- Multi-site: No

**Additional context**
Full error bump:
```sh
success source and transform nodes - 15.454s
success building schema - 10.577s
success createPages - 0.001s

 ERROR

Interface field Craft_FormInterface.fields expected but Craft_Form does not provide it.



  Error: Interface field Craft_FormInterface.fields expected but Craft_Form does not provide it.

  - graphql-runner.ts:105 GraphQLRunner.validate
    [jbrobbie-website]/[gatsby]/src/query/graphql-runner.ts:105:16

  - graphql-runner.ts:174 GraphQLRunner.query
    [jbrobbie-website]/[gatsby]/src/query/graphql-runner.ts:174:39

  - create-graphql-runner.ts:57
    [jbrobbie-website]/[gatsby]/src/bootstrap/create-graphql-runner.ts:57:8

  - create-pages-from-collection-builder.js:71 _callee$
    [jbrobbie-website]/[gatsby-plugin-page-creator]/create-pages-from-collection-builder.js:71:20

  - new Promise

  - create-pages-from-collection-builder.js:162 _createPagesFromCollectionBuilder
    [jbrobbie-website]/[gatsby-plugin-page-creator]/create-pages-from-collection-builder.js:162:44

  - create-pages-from-collection-builder.js:34 createPagesFromCollectionBuilder
    [jbrobbie-website]/[gatsby-plugin-page-creator]/create-pages-from-collection-builder.js:34:44


not finished createPagesStatefully - 4.526s

error Command failed with exit code 1.
```

**Comment by engram-design** [comment]
Are you able to try on [1.4.22](https://github.com/verbb/formie/releases/tag/1.4.22)? I was able to spin up a Gatsby site without any errors, but happy to have another look.

**Comment by webrgp** [comment]
Forgive me, I've pasted the Formie version from the other issue. The version I am using is the 1.4.22, and I've updated above so people don't get confused.

**Comment by engram-design** [comment]
Ah, gotcha. Looks like that is indeed an issue, I'll investigate shortly.

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.23",
  "...": "..."
}
```

Then run `composer update`. Do note that we've had to include a new config setting just for Gatsby. This is because there's a breaking change that needs to be made to make the changes for both Gatsby and regular GraphQL users.

Please add `'enableGatsbyCompatibility' => true,` to your `formie.php` config file - https://verbb.io/craft-plugins/formie/docs/get-started/configuration

**Comment by engram-design** [comment]
Fixed in [1.4.24](https://github.com/verbb/formie/releases/tag/1.4.24)
